storio-sqlite/src/main/java/com/pushtorefresh/storio/sqlite/operations/put/PutResult.java

```diff
@@ -127,7 +127,7 @@ public boolean wasNotInserted() {
      * even if all columns were same.
      */
     public boolean wasUpdated() {
-        return numberOfRowsUpdated != null;
+        return numberOfRowsUpdated != null && numberOfRowsUpdated > 0;
```

Comment:
It seems that we need it again (#423)
@artem-zinnatullin, I know you dislike it, but I think it is not true if we will return `wasUpdated() // true` if `0` rows were updated=)